Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/MSBuildTools.cs

```diff
@@ -26,7 +26,7 @@ public class MSBuildToolsConfig
         private int version = 0;
 
         [SerializeField]
```

Comment:
>SerializeField [](start = 9, length = 14)

For folks who are upgrading msbuildtool config, is it worth having FormerlySerializedAs, or is the delta here not something that really matters

Reply:
Added.